### PR TITLE
added solution 1 for problem_92

### PR DESCRIPTION
**Pull request by gamzeaksoy**
added solution 1 for problem_92

### CODE REVIEW

**Comment by cclauss** [comment]
Click `Details` below in ` Travis CI - Pull Request Failing after 2m — Build Failed     Details`

**Comment by itsvinayak** [comment]
I don't know why every time I push something it shows so many commits even though my branch is even with the master branch

**Comment by cclauss** [comment]
Please rebase this PR.  It currently contains 45 commits and solutions to both problem 92 and 99 and is therefore difficult to review.

File: project_euler/problem_92/sol1.py [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+Therefore any chain that arrives at 1 or 89 will become stuck in an endless loop. What is most amazing is that EVERY starting number will eventually arrive at 1 or 89.
+
+How many starting numbers below ten million will arrive at 89?
+"""
+
+#!/usr/bin/env python
+
+
+def get_digits(n):
+    return list(map(int, str(n)))
```
[the remark this answers is not in the document]
Comment (cclauss) [reply project_euler/problem_92/sol1.py]: Please add a doctest to this function also type hints `def get_digits(n: int) -> list:`

File: project_euler/problem_92/sol1.py [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+
+def classify(n):
+    return "".join(sorted(str(n)))
+
+
+terminators = {}
+
+
+def chain(start):
```
Comment (cclauss) [inline project_euler/problem_92/sol1.py]: Please add doctest and type hints.

File: project_euler/problem_92/sol1.py [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+"""
+
+#!/usr/bin/env python
+
+
+def get_digits(n):
+    return list(map(int, str(n)))
+
+
+def classify(n):
```
Comment (cclauss) [inline project_euler/problem_92/sol1.py]: Please add doctest and type hints.

File: project_euler/problem_92/sol1.py [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+        return False
+
+
+def main(starting_range: int, ending_range: int) -> int:
+    """
+    >>> main(1,10000000)
+    8581146
+    >>> main(2,100)
+    80
+    >>> main(-2,9)
```
Comment (cclauss) [inline project_euler/problem_92/sol1.py]: `>>> main(-2, 9)  # doctest: +IGNORE_EXCEPTION_DETAIL`

And then delete lines 67 through 76.

**Pull request closed without merging**